src/ShapeCrawler/ShapeCollection/SlideShapes.cs

```diff
@@ -140,11 +141,17 @@ public void AddPicture(Stream image)
                 new MagickReadSettings { BackgroundColor = MagickColors.Transparent });
 
             var originalFormat = imageMagick.Format;
-            if (!SupportedImageFormats.Contains(imageMagick.Format) || VectorImageFormats.Contains(imageMagick.Format))
+            if (!SupportedImageFormats.Contains(imageMagick.Format))
             {
                 imageMagick.Format = imageMagick.HasAlpha ? MagickFormat.Png : MagickFormat.Jpeg;
             }
 
+            if (VectorImageFormats.Contains(imageMagick.Format))
+            {
+                imageMagick.Format = MagickFormat.Png;
+                imageMagick.Density = new Density(384, DensityUnit.PixelsPerInch);
```

Comment:
Please comment this statement in WHY style like, e.g., `// in PowerPoint, the resolution of the rasterized version of SVG is 384 PPI` to avoid 384 being a magic number.